scenario_player/scenario.py

```diff
@@ -33,10 +34,19 @@ def __init__(self, yaml_path: pathlib.Path, data_path: pathlib.Path) -> None:
         self.scenario = ScenarioConfig(self._loaded)
         self.token = TokenConfig(self._loaded, data_path.joinpath("token.info"))
         self.spaas = SPaaSConfig(self._loaded)
+        self.validate()
 
         self.gas_limit = GAS_LIMIT_FOR_TOKEN_CONTRACT_CALL * 2
 
     @property
     def name(self) -> str:
         """Return the name of the scenario file, sans extension."""
         return self.path.stem
+
+    def validate(self):
+        # The amount of Tokens minted need to be greater than the amount of tokens
```

Comment:
That can go into a the method doc string, since it would be interesting to know for anyone looking up this method (once we render them somewhere).